### PR TITLE
Re-implement the member_uid searcher

### LINKED ISSUE #398: member_uid order
Related with https://github.com/coopdevs/timeoverflow/pull/354

Now is ordering by ID alphabetically but we need numerically order

I only see it in produccion, with more than 99 members :sweat_smile: 
fireshot capture 2 - banc del temps de gracia - https___www timeoverflow org_members

### PR DESCRIPTION
**Pull request by markets**
Re-implement the member_uid searcher by avoiding the column casting thing (gives some problems when sorting the "casted" column as an string). The new strategy: combine both fields in the controller by manupilating the query parameters sent to Ransack.

Closes #398 
Closes #400

### CODE REVIEW

**Comment by mllocs** [comment]
Hello @markets! One question, in the `schema.rb` it looks like the `member_uid` column is already an integer, shouldn't Ransack work out of the box like it does with other integer fields?

**Comment by markets** [comment]
Hi @mllocs, I'll try to explain this ... :sweat_smile: 

In current `develop` if you remove the `ransacker` (which makes the cast from integer to string), and you try to use the search feature it gives you this error:

```
PG::UndefinedFunction - ERROR:  operator does not exist: integer ~~* integer
LINE 1: ...."email" ILIKE '%asd%') OR "members"."member_uid" ILIKE 0)  ...
                                                             ^
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
```

Seems like `postgresql` doesn't support `ILIKE` filters over integer columns. So, in a first try, I implemented a `ransacker` to make a cast, but then this `ransacker` is used by Ransack also in the `sort_link` helper, so now it sorts the list "alphabetically" by a string :(

In summary, this new solution consists of: copy value from original `_cont` param to a new `_eq` param (`_eq` works for integers, since it doesn't use `ILIKE` under the hood), and combine them with `OR` before sending this params to Ransack.

**Comment by mllocs** [comment]
what about: https://github.com/activerecord-hackery/ransack/issues/765

**Comment by markets** [comment]
@mllocs cool, I'll make a try and send new PR, so we can compare and pick better solution. Thanks!

**Comment by markets** [comment]
@mllocs new proposal in #400
